### PR TITLE
If quantity is not stored on post, send Quasar quantity from signup

### PR DESCRIPTION
**Pull request by katiecrane**
#### What's this PR do?
When a user submits a post, we send the updated post to Quasar. However, while the quantity lives on the signup, they don't get that updated quantity! For now, let's send the quantity on the post only if it is not null, otherwise grab the quantity from the signup.

#### How should this be reviewed?
👀 

#### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Tested on staging.
- [ ] Pinged a PM if this is a larger PR that would benefit from some additional testing love.

### CODE REVIEW

File: app/Models/Post.php [hunk 1]
```diff
@@ -195,7 +195,7 @@ public function toQuasarPayload()
             'northstar_id' => $this->northstar_id,
             'type' => $this->type,
             'action' => $this->action,
-            'quantity' => $this->quantity,
+            'quantity' => $this->quantity ? $this->quantity : $this->signup->quantity,
```
Comment (sbsmith86) [inline app/Models/Post.php]: I don't think I understand this issue. Why would we ever want to send the signup quantity (which will be the total quantity across all posts) here?

Comment (katiecrane) [reply app/Models/Post.php]: @sbsmith86 because as of right now we are not updating the quantity column on the post when a post is submitted

**Comment by katiecrane** [comment]
@DFurnes ah we have that on the signup! I figured that since we are trying to turn on quantity on the post really soon that we won't need this check for long

**Comment by DFurnes** [comment]
Yeah, that's totally fair too. 👍

**Comment by sbsmith86** [comment]
@katiecrane should hide this logic under the feature flag like we do in `signup->getQuantity` ?

**Comment by katiecrane** [comment]
@sbsmith86 I was wondering that too! I wasn't sure if it was totally necessary since if that flag is on, then the quantity on the post will not be `null` and we'll still get what we want. Probably safer to do it that way though, I'll bust it out into a method that does that so we can rest easy

**Pull request merged**